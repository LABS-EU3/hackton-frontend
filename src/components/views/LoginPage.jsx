import React from "react";
import image from "../../assets/Signup.png";
import { Unboarding } from "../templates";

const LoginPage = () => (
  <Unboarding
    ctaText="Log In"
    imageType={image}
    imageText="Log In now!!"
    formHeader="Log into Portal"
    formParagraph="We missed you, you missed us now get back into action."
  />
);

export default LoginPage;
