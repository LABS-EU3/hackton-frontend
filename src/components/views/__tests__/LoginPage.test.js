import React from 'react'
import { Router, Route } from 'react-router';
import { createMemoryHistory } from 'history';
import { render } from '@testing-library/react'
import LoginPage from '../LoginPage';
const history = createMemoryHistory()

const renderWithRouter = Component => render(
  <Router history={history}>
    <Route component={Component} />
  </Router>
)

describe("It should render dashboard page correctly", () => {
    // let wrapper;
  
    // beforeEach(() => {
    //   wrapper = shallow(renderWithRouter(LoginPage));
    // });
  
    it("renders the LoginPage component correctly", () => {
    const render = () => renderWithRouter(LoginPage)
      expect(render).toMatchSnapshot();
    });
  });