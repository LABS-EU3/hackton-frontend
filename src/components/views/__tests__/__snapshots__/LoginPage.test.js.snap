// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render dashboard page correctly renders the LoginPage component correctly 1`] = `[Function]`;
